app/jobs/membership_event_job.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Documentation: https://developer.github.com/v3/activity/events/types/#membershipevent
+class MembershipEventJob < ApplicationJob
+  queue_as :github_event
+  def perform(payload_body)
+    return true unless payload_body["action"] == "removed"
+
+    github_user_id = payload_body.dig("member", "id")
+    github_organization_id = payload_body.dig("organization", "id")
+
+    organization = Organization.find_by(github_id: github_organization_id)
+    user = organization.users.find_by(id: github_user_id)
```

Comment:
It should be `user = organization.users.find_by(uid: github_user_id)` In the database uid is the github_id.